components/commons/fonts.css

```diff
@@ -27,15 +27,15 @@ governing permissions and limitations under the License.
   --spectrum-font-family-condensed: adobe-clean-condensed, var(--spectrum-font-family-base);
   */
 
-  --spectrum-font-family-base: adobe-clean-ux, adobe-clean, "Source Sans Pro", var(--spectrum-font-fallbacks-sans);
-  --spectrum-font-family-han: adobe-clean-han-japanese, var(--spectrum-font-fallbacks-sans);
-  --spectrum-font-family-ar: myriad-arabic, var(--spectrum-font-fallbacks-sans);
-  --spectrum-font-family-he: var(--spectrum-font-fallbacks-sans);
-  --spectrum-font-family-zh: var(--spectrum-font-family-han);
-  --spectrum-font-family-zhhans: var(--spectrum-font-family-han);
-  --spectrum-font-family-ko: var(--spectrum-font-family-han);
-  --spectrum-font-family-ja: var(--spectrum-font-family-han);
-
+  --spectrum-font-family-base: 'adobe-clean-ux', 'adobe-clean', 'Source Sans Pro', var(--spectrum-font-fallbacks-sans);
+  --spectrum-font-family-han: 'adobe-clean-han-japanese', var(--spectrum-font-fallbacks-sans);
+  --spectrum-font-family-ar: 'adobe-arabic', 'myriad-arabic', var(--spectrum-font-fallbacks-sans);
+  --spectrum-font-family-he: 'adobe-hebrew', var(--spectrum-font-fallbacks-sans);
+  --spectrum-font-family-zhhans: 'adobe-clean-han-simplified-c', 'SimSun', 'Heiti SC Light', var(--spectrum-font-fallbacks-sans);
+  --spectrum-font-family-zh: var(--spectrum-font-family-zhhans);
+  --spectrum-font-family-zhhant: 'adobe-clean-han-traditional', 'MingLiu', 'Heiti TC Light', var(--spectrum-font-fallbacks-sans);
```

Comment:
Looking at MingLiu, I'd prefer we have a gothic option here instead. I looks like we get 'JhengHei UI' and 'JhengHei' installed from the Microsoft Store when we enable Chinese Typography in Windows 10 (although it's not clearly documented :-/ ). 

![image](https://user-images.githubusercontent.com/5404261/63457462-f709b580-c405-11e9-8837-b3aadc62b6d8.png)

Reply:
Thanks @tappily! Are you suggesting we change it to:

```
--spectrum-font-family-zhhant: adobe-clean-han-traditional', 'JhengHei UI', 'JhengHei', 'MingLiu', 'Heiti TC Light', var(--spectrum-font-fallbacks-sans);
```

Let me know and I'll make the change.

Reply:
Close. I'd prefer MingLui not be in this list at all. MingLui works for the article/ serif style, but not here.

Reply:
@tappily ok, updated! Are we good to go?

Reply:
Sorry, i missed the MS prefix. needs to be "Microsoft JhengHei UI"  and "Microsoft JhengHei"